index.html

```diff
@@ -5,8 +5,58 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Weather Report</title>
-</head>
+    <link href="styles/index.css" rel="stylesheet" type="text/css"/>
+    <link id="vibe" href="styles/vibe2.css" rel="stylesheet" type="text/css"/>
+
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <<link href="https://fonts.googleapis.com/css2?family=Baloo+Bhaijaan+2&family=Bebas+Neue&family=Cormorant+Garamond:ital@1&family=Fuzzy+Bubbles:wght@700&family=Gwendolyn:wght@700&family=Londrina+Shadow&family=Readex+Pro&display=swap" rel="stylesheet">
```

Comment:
Watch out for the extra `<` typo here. And make sure to close the `<head>` tag with `</head>` before opening the `<body>`.